pkg/kubelet/network/cni/cni.go

```diff
@@ -273,7 +273,7 @@ func (plugin *cniNetworkPlugin) deleteFromNetwork(network *cniNetwork, podName s
 	netConf, cniNet := network.NetworkConfig, network.CNIConfig
 	glog.V(4).Infof("About to del CNI network %v (type=%v)", netConf.Name, netConf.Plugins[0].Network.Type)
 	err = cniNet.DelNetworkList(netConf, rt)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
PLEASE PLEASE comment why this is acceptable.

Reply:
@thockin Addressed. PTAL. Thanks.